.github/workflows/stress-test.yml

```diff
@@ -0,0 +1,35 @@
+# Nightly Stress Test for self-hosted runners
+name: Self-hosted Runners Nightly Stress Test
+on:
+  schedule:
+    # Triggers at 11pm every night.
```

Comment:
Small nit, can you leave the timezone in the comment to make it obvious for anyone reviewing

Reply:
Added the timezone comment. It's in UST so I changed the time to make it run at 11pm PST.